rxbinding-leanback-v17/src/androidTest/java/com/jakewharton/rxbinding/support/v17/leanback/widget/RxSearchBarTestActivity.java

```diff
@@ -0,0 +1,21 @@
+package com.jakewharton.rxbinding.support.v17.leanback.widget;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.support.v17.leanback.widget.SearchBar;
+import android.support.v17.leanback.widget.SearchEditText;
+import com.jakewharton.rxbinding.support.v17.leanback.R;
+
+public final class RxSearchBarTestActivity extends Activity {
+  SearchBar searchBar;
+  SearchEditText searchEditText;
+
+  @Override protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+
+    setContentView(R.layout.lb_search_fragment);
+
+    searchBar = (SearchBar) findViewById(R.id.lb_search_bar);
```

Comment:
I can't `new SearchBar(this)` here. `searchEditText ` will be `null`. [In `SearchBar` class, mSearchTextEditor is created in `onFinishInflate`](https://github.com/android/platform_frameworks_support/blob/master/v17/leanback/src/android/support/v17/leanback/widget/SearchBar.java#L182).